internal/host/transition_test.go

```diff
@@ -5331,6 +5460,21 @@ func generateMajorityGroup(machineNetworks []*models.MachineNetwork, hostId strf
 	return string(tmp)
 }
 
+func addL3Connectivity(majorityGroupsStr string, hostId strfmt.UUID) string {
+	majorityGroups := make(map[string][]string)
+	if majorityGroupsStr != "" {
+		if err := json.Unmarshal([]byte(majorityGroupsStr), &majorityGroups); err != nil {
+			return ""
```

Comment:
In tests I'd think this kind of thing would fail the test rather than return silently.

Instead of a conditional here can we do `Expect(json.Unmarshal([]byte(majorityGroupsStr), &majorityGroups)).To(Succeed())`?
Same for the `Marshal` call below this.

Reply:
OK